docsbox/docs/views.py

```diff
@@ -240,31 +326,35 @@ def get(task_id):
         except Exception as e:
             return abort(500, e, request, traceback=traceback.format_exc())
 
-        return response
+        return response.serialize()
 
 
-def get_file_info(file_id, filename="", save_file=False):
+def get_file_info(file_id: str, filename="", save_file=False):
+    file_info = FileInfo()
     if db.exists('fileId:' + file_id) == 0:
-        file_info = {"file_id": file_id, "mimetype": "", "filename": filename, "datetime": datetime.now().strftime('%Y/%m/%d-%H:%M:%S')}
+        file_info.file_id = file_id
+        file_info.mimetype = ""
+        file_info.filename = filename
+        file_info.datetime = datetime.now().strftime('%Y/%m/%d-%H:%M:%S')
         if save_file:
-            file_info["file_path"] = store_file_from_id(file_id, filename)
-            file_info["mimetype"], file_info["pdf_version"] = get_file_mimetype(file_info["file_path"])
+            file_info.file_path = store_file_from_id(file_id, filename)
+            file_info.mimetype, file_info.pdf_version = get_file_mimetype(file_info.file_path)
         else:
-            file_info["mimetype"], file_info["pdf_version"] = get_file_mimetype_from_id(file_id, filename)
+            file_info.mimetype, file_info.pdf_version = get_file_mimetype_from_id(file_id, filename)
         db.set('fileId:' + file_id, json.dumps(file_info))
     else:
         file_info = json.loads(db.get('fileId:' + file_id))
         updated = False
 
-        if file_info["filename"] is None and filename:
-           file_info["filename"] = filename
-           updated = True
+        if file_info.filename is None and filename:
+            file_info.filename = filename
```

Comment:
extra space on this and some lines below?

Reply:
It is okay, for some reason it had 3 spaces instead of the normal 4 spaces so I corrected it